routes/licenses.test.js

```diff
@@ -11,12 +11,20 @@ describe('license routes', () => {
   const services = { licenseStore, fileData, licenses };
   const licensesMock = [
```

Comment:
We're probably at the point where we should have a factory for that 😹

Reply:
🏭

Reply:
🤔 Should it use the model constructor or is it fine to just return an object? The latter is probably faster, right?